tests/indices/list/test_base.py

```diff
@@ -187,11 +209,58 @@ def test_query(
     assert node_info["end"] == 12
 
 
+def mock_text_splitter(
+    prompt: Prompt, num_chunks: int, padding: Optional[int] = 1
+) -> TokenTextSplitter:
+    """Mock text splitter."""
```

Comment:
if this isn't used elsewhere you could just define this within test_index_overlap. Use a context manager within the function to patch e.g. 
```
with patch.object(PromptHelper, "...", side_effect=_mock_text_splitter_with_space)

```

Also defining it as _mock_text_splitter_with_space probably clearer